api/src/main/java/org/openmrs/api/impl/PersonServiceImpl.java

```diff
@@ -564,72 +560,59 @@ public RelationshipType saveRelationshipType(RelationshipType relationshipType)
 	 * @see org.openmrs.api.PersonService#getPersonAttributeTypes(org.openmrs.util.OpenmrsConstants.PERSON_TYPE,
 	 *      org.openmrs.api.PersonService.ATTR_VIEW_TYPE)
 	 */
+	
+	private String userPatientAttributes(String patientAttributeProperty, String userAttributeProperty,
```

Comment:
also please move this private method below the `getPersonAttributeTypes` where its used the first time. this way it reads more naturally from top to bottom.